docs/UserGuide.md

```diff
@@ -350,7 +350,10 @@ Unsuccessful Output:
 
 - Invalid command: `Invalid command.` + help message with list of all commands
 
-- Missing arguments: `Invalid syntax: Missing arguments. Syntax: find-plan FRIEND_NAME`
+- Missing arguments: `Invalid command format!
```

Comment:
for this one, victor used "invalid syntax" in add-plan instead
![Screenshot 2023-11-02 at 1 16 03 AM](https://github.com/AY2324S1-CS2103T-W16-4/tp/assets/110958143/83997d6a-8955-46a5-81de-8e717c09ba0f)

Reply:
Hmmm... I set it as invalid command format since find-friend does the same thing...